server/coin-gecko-scraper.js

```diff
@@ -0,0 +1,60 @@
+const rp = require('request-promise');
```

Comment:
@Shapokhan good to see that you achieved in just about 60 LOC (lines of code) pretty much the same functionality that [i did in about 90 LOC](https://github.com/omeryousaf/sasta-miner/blob/334d339d26dcb618a064299c83e9583e7f8105f2/server/coin-scraper.js). 👍

Reply:
I get more confidence when you appreciate me. Thank you.